plugins/inara.py

```diff
@@ -50,6 +50,9 @@
 this.fleet = None
 this.shipswap = False	# just swapped ship
 
+this.last_update_time = time.time()  # last time since we updated (set to now because we're going to update quickly)
```

Comment:
That ' since' doesn't belong there.

Reply:
Whoops! thats from me changing stuff around. will fix shortly